MapboxMobileEvents/Categories/CLLocation+MMEMobileEvents.m

```diff
@@ -24,6 +24,20 @@ - (CLLocationAccuracy)mme_roundedVerticalAccuracy {
     return round(self.verticalAccuracy);
 }
 
+- (CLLocationAccuracy)mme_roundedSpeedAccuracy {
+    if (@available(iOS 13.4, *)) {
+        return round(self.speedAccuracy);
+    }
+    return 0;
+}
+
+- (CLLocationAccuracy)mme_roundedCourseAccuracy {
```

Comment:
https://github.com/mapbox/mapbox-events-ios/pull/203/files#diff-bd4c68f875c0aa434bb834c6903bac29R15

I'm wondering if there is precedent for when to expose functions? Vs when to default them?

I see this CoreLocation method isn't usable if not iOS 13.4+? I understand 0 accuracy means it's not accurate. But I also wonder if the existence (not available vs 0) would lead to more confusion?
```
@available(iOS 13.4, *)
- (CLLocationAccuracy)mme_roundedSpeedAccuracy
```

Reply:
This is a good point. Since this is not a required attribute, marking availability makes more sense.